### ISSUE TITLE
Error: Method not found: 'ButtonTheme.bar'.                   child: ButtonTheme.bar(

### ISSUE THREAD

**Issue by Marmik123** [rodydavis/data_tables#25]

**Comment by Marmik123** [comment]
I am getting this error while running on flutter web and in the error message path it is showing error in stateless_datatable.dart file.

**Comment by Marmik123** [comment]
![image](https://user-images.githubusercontent.com/73093114/108840112-e722f100-75fb-11eb-8fec-fd0cafd9e7a6.png)

**Comment by rodydavis** [comment]
Fixed on the prerelease

**Comment by yingshaoxo** [comment]
> Fixed on the prerelease

Thanks.

```
  data_tables:
    git:
      url: https://github.com/rodydavis/data_table
```

**Comment by rodydavis** [comment]
fixed with version `1.2.0`
